packages/providers/upload-rackspace/README.md

```diff
@@ -43,6 +45,11 @@ module.exports = ({ env }) => ({
         region: env('RACKSPACE_REGION'),
         container: env('RACKSPACE_CONTAINER'),
       },
+      actionOptions: {
+        upload: {},
+        uploadStram: {},
```

Comment:
You wrote `uploadStram` instead of `uploadStream` :p Same in the other providers

Reply:
🤦🏼 Fixed in f1ee0ff0ec3301e0695cc1e781116524736f2f9a